doc/source/changelog.rst

```diff
@@ -22,6 +21,8 @@ MontePy Changelog
 * Made ``Material.is_atom_fraction`` settable (:issue:`511`). 
 * Made NumberedObjectCollections act like a set (:issue:`138`).
 * Automatically added children objects, e.g., the surfaces in a cell, to the problem when the cell is added to the problem (:issue:`63`).
+* Added ability to parse all MCNP objects from a string (:pull:`595`).
+* Added function: :func:`~montepy.mcnp_problem.MCNP_Problem.parse` to parse arbitrary MCNP object (:pull:`595`).
```

Comment:
Is there a reason these link to a `:pull:` and not an `:issue:`?

Reply:
Yeah not a good reason. I thought maybe the PR might give more info, but I think it's not a ton of info here. I'll change it.